.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: ruby
+sudo: false
 
 rvm:
-  - 2.0.0
-  - 1.9.3
+  - 2.3.1
 
 gemfile:
   - Gemfile
   - Gemfile_ar30
   - Gemfile_ar31
   - Gemfile_ar32
-  - Gemfile_ar40threadsafe
-  - Gemfile_ar42threadsafe
-
-sudo: false
-
-before_script: echo $OLD_RAKE
+  - Gemfile_ar40
```

Comment:
I believe it needs to be specified here and then the matrix acts as an override. The matrix is implicitly generated from the combination of Ruby versions, Gemfiles, etc.